src/sync.rs

```diff
@@ -12,7 +20,31 @@ pub(crate) use loom::thread::yield_now;
 pub(crate) mod atomic {
     #[cfg(target_pointer_width = "64")]
     pub use core::sync::atomic::AtomicU64;
-    pub(crate) use core::sync::atomic::{fence, AtomicIsize, AtomicPtr, AtomicUsize};
+    pub(crate) use core::sync::atomic::{AtomicIsize, AtomicPtr, AtomicUsize, Ordering};
+
+    pub fn light_barrier() {
+        #[cfg(miri)]
```

Comment:
Why not cfg the whole function definition?

Reply:
That certainly is nicer, updated :+1: